spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -99,5 +99,14 @@
         )
       end
     end
+
+    context 'When the user is VIP' do
+      let(:user) { create(:user, vip: true) }
+      let!(:user_targets) { create_list(:target, 5, user:) }
```

Comment:
```suggestion
      let!(:user_targets) { create_list(:target, Target::MAX_TARGETS, user:) }
```
I think it should use the MAX_TARGETS constant